sergiolucaco/gympad
# Setup boiler plate of the application

## Description
This PR brings the boiler plate for the react application created with the create-react-app package and brings the `commitizen` package to trigger the `commit` script (check pacgae.json script section) as well as the `commitlint` functionality in case the commit is done in the traditional way. 

## Related tickets
https://github.com/sergiolucaco/gympad/projects/1#card-16195200
https://github.com/sergiolucaco/gympad/projects/1#card-16193789
https://github.com/sergiolucaco/gympad/projects/1#card-16194117

## Testing
- Checkout the branch
- `npm install`
- `npm start` (check `localhost:3000`, should open automagically)
- Try to commit with `git commit -m` and with `npm run commit`
- Verify everything works
